### DISCUSSION TITLE
Is it possible to recover an account of the account password and the account recovery are lost?

### DISCUSSION THREAD

**Discussion by gonzalob-ogovsystem** [globaleaks/globaleaks-whistleblowing-software#4557] [Q&A] [answered]
Hi all, anyone can help to reset admin account password, i dont have the password and dont have the recovery key.

Is there any change to reset the admin password ?

Thanks in advance

**Comment by evilaliv3** [comment, marked as answer]
Thank you for your question @gonzalob-ogovsystem 

Actually globaleaks implements secure encryption and the only possibility in your case to restore the account is to ask an other admin is to ask an other administrator.

If no other administrators access to this account is lost and with it access to the key escrows that protects recipients accounts.

Is this system used in production? Do you have submissions on the system?

**Comment by gonzalob-ogovsystem** [comment]
Hi evilaliv3, thank for you fast answer.

Only 1 account is admin , and is the one that i tried to recover.

Yes, the system is in production, and yes i have full access to the system (via ssh)

Thanks for all.

**Comment by evilaliv3** [comment]
If you are the only administrator there is unfortunately no way to recover access to the account and the included key escrows that protects in case of recipient password loss.

My advice is to get back to recipients asking to export every report and then to create a new system.

**Comment by gonzalob-ogovsystem** [comment]
Thanks evilaliv3, i just created new site and all works as expected.

**Comment by evilaliv3** [comment]
Thank you for your feedback @gonzalob-ogovsystem.

Closing the ticket then.
